toolbox/commands.py

```diff
@@ -0,0 +1,135 @@
+import typing as t
```

Comment:
This file should be `command_choices.py`

Reply:
My idea was that other utilities relating to commands could also go here.